.vscode/launch.json

```diff
@@ -1,24 +1,24 @@
 {
   "version": "0.2.0",
   "configurations": [
-      {
-          "name": "Debug on Windows",
-          "type": "cppvsdbg",
-          "request": "launch",
-          "program": "${workspaceFolder}/build/<path-to-bin-file>",
-          "args": [],
-          "stopAtEntry": false,
-          "cwd": "${workspaceFolder}",
-          "environment": [],
-          "externalConsole": false
-      },
-      {
-          "name": "Debug on Linux",
-          "type": "gdb",
-          "request": "launch",
-          "target": "${workspaceFolder}/bazel-bin/<path to the bin file>",
-          "cwd": "${workspaceRoot}",
-          "valuesFormatting": "parseText"
-      }
+    {
+      "name": "Debug on Windows",
+      "type": "cppvsdbg",
+      "request": "launch",
+      "program": "${workspaceFolder}/build/exporters/ostream/Debug/ostream_log_test.exe",
```

Comment:
Is this build path from bazel, and need manual update for CMake build?

Reply:
Sorry.this file is committed by mistake.I will restore it tomorrow.